server/server.js

```diff
@@ -0,0 +1,30 @@
+// Server
+const express = require('express')
+const app = express()
+const port = 5000
+const path = require('path');
+
+
+app.listen(port, () => {
+  console.log(`Example app listening on port ${port}`)
+})
+
+// Pages
+app.get('/', (req, res) => {
+  res.sendFile(path.join(__dirname, '../client/index.html'))
```

Comment:
Server side is displaying HTML pages, whereas I would expect "simple" JSON objects.